main.nf

```diff
@@ -595,7 +602,7 @@ if(params.aligner == 'bwameth'){
         input:
         set val(name), file(reads) from trimmed_reads
         file bwa_meth_indices from bwa_meth_indices.collect()
-        file wherearemyfiles
+        file wherearemyfiles from wherearemyfiles_2
```

Comment:
Same here, please use explicit names for channels :-)
It gets super confusing at some point otherwise....